sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,302 @@
+    -- Do we need to include any other events for computing "all_visits"?
+    AND (
+      category = 'newtab'
+      AND name IN ('opened')
+      OR category = 'newtab.search'
+      AND name IN ('issued')
+      OR category = 'newtab.search.ad'
+      AND name IN ('click')
+      OR category = 'pocket'
+      AND name IN (
+        'click',
+        'impression',
+        'dismiss',
+        'thumb_voting_interaction',
+        'thumb_voting_interaction'
+      )
+      OR category = 'topsites'
+      AND name IN ('click', 'impression', 'dismiss')
+      OR category = 'newtab'
+      AND name IN (
+        'weather_change_display',
+        'weather_open_url_provider',
+        'weather_location_selected'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'wallpaper_click',
+        'wallpaper_category_click',
+        'wallpaper_highlight_cta_click',
+        'wallpaper_highlight_dismissed'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'topic_selection_open',
+        'topic_selection_dismiss',
+        'topic_selection_topics_saved'
+      )
+      OR category = 'newtab'
+      AND name IN (
+        'sections_block_section',
+        'sections_follow_section',
+        'sections_unblock_section',
+        'sections_unfollow_section'
+      )
+      OR category = 'newtab'
+      AND name IN ('inline_selection_click')
+    )
+)
+SELECT
+  submission_date,
+  client_id,
+  mozfun.map.get_key(event_details, 'newtab_visit_id') AS newtab_visit_id,
+  -- ANY_VALUE: because we do not expect those fields to differ within a visit
+  ANY_VALUE(app_version) AS app_version,
+  ANY_VALUE(os) AS os,
+  ANY_VALUE(channel) AS channel,
+  ANY_VALUE(locale) AS locale,
+  ANY_VALUE(browser_version) AS browser_version,
+  ANY_VALUE(country) AS country,
+  ANY_VALUE(newtab_homepage_category) AS newtab_homepage_category,
+  ANY_VALUE(newtab_newtab_category) AS newtab_newtab_category,
+  ANY_VALUE(organic_content_enabled) AS organic_content_enabled,
+  ANY_VALUE(sponsored_content_enabled) AS sponsored_content_enabled,
+  ANY_VALUE(sponsored_topsites_enabled) AS sponsored_topsites_enabled,
+  ANY_VALUE(organic_topsites_enabled) AS organic_topsites_enabled,
+  ANY_VALUE(newtab_search_enabled) AS newtab_search_enabled,
+  -- MAX: to give precedence to "true" if there are multiple outcomes within a visit
+  MAX(
+    CASE
+      WHEN (
+          event_category = 'newtab'
+          AND event_name = 'opened'
+          AND (
+            mozfun.map.get_key(event_details, 'source') = 'about:home'
+            AND newtab_homepage_category = 'enabled'
+          )
+          OR (
+            mozfun.map.get_key(event_details, 'source') = 'about:newtab'
+            AND newtab_newtab_category = 'enabled'
+          )
+        )
+        THEN TRUE
+      ELSE FALSE
+    END
+  ) AS is_default_ui,
+  MAX(IF(event_category = 'newtab' AND event_name = 'opened', TRUE, FALSE)) AS is_newtab_opened,
```

Comment:
remove unnecessary IF

Reply:
@jaredsnyder `LOGICAL_OR` was a good suggestion. Thanks!
I can remove both max and if